external-crates/move/move-compiler/src/hlir/ast.rs

```diff
@@ -1411,18 +1411,6 @@ impl AstDebug for BuiltinFunction_ {
     }
 }
 
-impl AstDebug for ExpListItem {
-    fn ast_debug(&self, w: &mut AstWriter) {
-        match self {
-            ExpListItem::Single(e, st) => w.annotate(|w| e.ast_debug(w), st),
-            ExpListItem::Splat(_, e, ss) => {
-                w.write("~");
-                w.annotate(|w| e.ast_debug(w), ss)
-            }
-        }
-    }
-}
-
```

Comment:
Do you need to add parens to the debug for call/builtin call?

Reply:
It all seems more or less coherent how it's printed out right now. Both forms already surrounded their forms with `()`, so this actually removes some extra `()`s that were getting stacked on.